README.md

```diff
@@ -107,7 +107,7 @@ $ echo "PATH=<LocationOfDUNES>:$PATH" >> .bashrc
 If you want to rebuild the DUNES image pick your preferred terminal application and input the following command:
 
 ```console
-<PathToDUNES>/DUNES$ ./bootstrap.sh
+<PathToDUNES>/DUNES$ python3 ./bootstrap.py
```

Comment:
If bootstrap.py has an "execute" permission may be it would be better to remove python3 here?

Reply:
I think we should probably leave it here for windows users? It doesn't hurt to have it, I don't think.